packages/client/src/components/OrderDataCompletion/OrderDataCompletion.tsx

```diff
@@ -0,0 +1,88 @@
+import {
+  Box,
+  Button,
+  Container,
+  FormControl,
+  FormControlLabel,
+  Grid,
+  InputLabel,
+  MenuItem,
+  Radio,
+  RadioGroup,
+  Select,
+  TextField,
+  Typography,
+} from '@mui/material';
+
+const OrderDataCompletion = () => {
+  return (
+    <Container fixed>
+      <Box sx={{ bgcolor: '#FAFAFA' }}>
+        <Grid item container spacing={5}>
+          <Grid item xs={6} xl={6}>
+            <Box marginLeft={5}>
+              <Typography variant='h5' color='primary.main' marginBottom={3}>
+                Godzina dostawy
+              </Typography>
+              <FormControl sx={{ width: '60%' }}>
+                <InputLabel id='demo-simple-select-label'>Godzina dostawy</InputLabel>
+                <Select labelId='demo-simple-select-label' id='demo-simple-select' label='hours' size='small'>
```

Comment:
```suggestion
                <Select labelId='demo-simple-select-label' id='demo-simple-select' label='hours' size='small' value=''>
```

Naprawia ostrzeżenie, które pokazuje się przy obecnej wersji w konsoli.